src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs

```diff
@@ -11,8 +11,16 @@ namespace Microsoft.Health.SqlServer.Features.Schema
     [JsonConverter(typeof(StringEnumConverter))]
     public enum SchemaVersionStatus
     {
-        Started,
-        Completed,
-        Failed,
+        // Note - Updating SchemaVersionStatus to lower case to be consistent with the statuses applied when schema AutomaticUpdates is set to true
```

Comment:
nit: schema AutomaticUpdates -> SchemaOptions:AutomaticUpdatesEnabled?

Reply:
Yep! Added the exact name